mappings/net/minecraft/client/render/entity/model/WitherEntityModel.mapping

```diff
@@ -1,2 +1,15 @@
 CLASS net/minecraft/class_621 net/minecraft/client/render/entity/model/WitherEntityModel
+	FIELD field_27532 root Lnet/minecraft/class_630;
+	FIELD field_27533 centerHead Lnet/minecraft/class_630;
+	FIELD field_27534 rightHead Lnet/minecraft/class_630;
+	FIELD field_27535 leftHead Lnet/minecraft/class_630;
+	FIELD field_27536 ribcage Lnet/minecraft/class_630;
+	FIELD field_27537 tail Lnet/minecraft/class_630;
+	METHOD <init> (Lnet/minecraft/class_630;)V
+		ARG 1 root
+	METHOD method_32066 rotateHead (Lnet/minecraft/class_1528;Lnet/minecraft/class_630;I)V
+		ARG 0 entity
```

Comment:
```suggestion
		ARG 0 wither
```

Reply:
The entity parameters in the models are named `entity`. We don't name them after the type. (that's what the method's parameter types are for)